akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -395,10 +396,22 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef, joinConfigCompatCh
       context.actorOf(propsWithDispatcher, name = "downingProvider")
     }
 
-    if (seedNodes.isEmpty)
-      logInfo("No seed-nodes configured, manual cluster join required")
-    else
+    if (seedNodes.isEmpty) {
+      if (isClusterBootstrapUsed)
+        logDebug("Cluster Bootstrap is used for joining")
+      else
+        logInfo("No seed-nodes configured, manual cluster join required")
```

Comment:
Perhaps we should refer to Cluster Bootstrap in this message?

Reply:
I don't think it would be applicable to log something unused in this case.